README.md

```diff
@@ -111,54 +111,4 @@ The `price` and `mtsCreate` timestamp must both be provided to all update handle
 
 ### Executing on Live Markets
 
-To run a strategy against the live marketplace, attach a `WSv2` instance from `bitfinex-api-node` to the strategy object on `ws` and call the `exec` method to bind the Bitfinex API listeners to the various strategy update methods:
-
-```js
-const debug = require('debug')('bfx:hf:strategy:example:exec')
-const HFS = require('bfx-hf-strategy')
```

Comment:
Whats the reason for removing this? It seems to me like its all still valid?

Reply:
This repo no longer provides the `exec` method, since it's been extracted into `bfx-hf-strategy-exec`, so I thought linking to the readme there which shows example usage is a good fix.

Reply:
Ok cool